fili-core/src/main/resources/moduleConfig.properties

```diff
@@ -53,6 +53,10 @@ bard__memcached_expiration_seconds = 3600
 # Data Cache
 bard__druid_cache_enabled = true
 
+# Maximum Druid response size to cache, in bytes. Defaults to MAX_LONG
+# If using memcached, make sure this aligns with the setting memcached is using, overridden with it's -I parameter
+bard__druid_max_response_length_to_cache = 9223372036854775807
```

Comment:
There are 2 ways to default a property:

1. Have the default be set in the module config that the property belongs to
2. Set the default every place the code accesses the property

Since we can't easily know every place that might access the property, it's safer and easier to manage to set the default here in the module properties. That said, we _do_ indicate that the default is `MAX_LONG` 2 lines up from here, but it may not be clear that the huge number _is_ a long, so I've added that number after in parenthesis to make it more clear.